5001-6000/LIT5196MagicPrayer.xml

```diff
@@ -35,7 +36,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p>Generic ID for magic prayers to gain respect and favour, usually accompanied by <foreign xml:lang="gez">ግርማ፡ ሞገስ፡</foreign>
```

Comment:
I don't understand what you mean by "accompanied by `<foreign xml:lang="gez">ግርማ፡ ሞገስ፡</foreign>`". Do you mean that the words ግርማ፡ ሞገስ፡ are usually part of the incipit?